Create presenting.md (#2591)
alan-turing-institute/the-turing-way: book/website/community-handbook/presenting.md

> # Presenting About the Turing Way

Reviewer: ```suggestion
# Presenting About _The Turing Way_
```

> Talks are a great way to engage new communities, teach people about open and reproducible practices, and expand the reach of the Turing Way by introducing the book and inviting new contributors. We encourage any Turing Way contributor to present on behalf of the Turing Way, and we ask that you use the guidelines detailed below to shape your talk and ensure consistency with TTW style. 

Reviewer: ```suggestion
Talks are a great way to engage new communities, teach people about open and reproducible practices, and expand the reach of _The Turing Way_ by introducing the book and inviting new contributors. We encourage any Turing Way contributor to present on behalf of _The Turing Way_, and we ask that you use the guidelines detailed below to shape your talk and ensure consistency with _The Turing Way_ style. 
```

Reviewer: The Turing Way is the official name and it must be in italics. I would not use TTW as this is unclear to anyone outside of the core team.

> 

Reviewer: Could we add in her a section on: 
### Suggesting venues for talks
If you would like _The Turing Way_ to talk at your event or at a community meeting, please do get in touch with our Community Manager or promote the opportunity on our #community Slack channel.

### Planning a Talk

> Whether you are submitting an abstract to a conference or you have been invited to speak, a great thing to do is to engage the Turing Way community for support and to publicize the event. Even if the talk is not open to all, you can work with other community members to refine your slides and give a practice talk if you'd like. Please post the details in Slack under the ______ channel.

Reviewer 2: Reminder to fill in the Slack channel name when this has been established

Author: @aleesteele did we pick a channel for this, or do we want to create a new one?

Reviewer: we could use the community channel as I think that is the one that most talks and opportunities like that are put in now.

Reviewer 3: Sorry I'm late to this! Seems like there are three applicable channels here:
#general : usually used to ask questions that aren't 
#community : where community celebrations, news, and organisational-related
#events : used to promote external events (in the wider open science network, etc.)

Would agree that for now, the community channel makes sense. Am currently reviewing the slack channels, so names may change at some point in the future to account for folk's needs: https://docs.google.com/spreadsheets/d/11MmzROZGpSkzoB01ATOTpklXrIg86eKSm1xUmPZuhlA/edit#gid=547657140

Reviewer 3: Following up on this, I don't think including slack happens naturally within the community. How about using the wording "If you'd like, please post the details in Slack in the #community channel"?

> Whether you are submitting an abstract to a conference or you have been invited to speak, a great thing to do is to engage the Turing Way community for support and to publicize the event. Even if the talk is not open to all, you can work with other community members to refine your slides and give a practice talk if you'd like. Please post the details in Slack under the ______ channel.

Reviewer: ```suggestion
Whether you are submitting an abstract to a conference or you have been invited to speak, a great thing to do is to engage _The Turing Way_ community for support and to publicize the event. Even if the talk is not open to all, you can work with other community members to refine your slides and give a practice talk if you'd like. Please post the details in Slack under the ______ channel.
```

### Preparing Slides

> Please create a GitHub issue using the ______ template (link), which provides links to the resources you will need as well as a checklist for preparing your talk. You will find a range of presentation templates, as well as past presentations, in the Promotion Pack folder on Google Drive. Feel free to download the templates and use the slides as a guide for style and content.

Reviewer 2: ```suggestion
Please create a GitHub issue using the ["Give a talk" issue template](https://github.com/alan-turing-institute/the-turing-way/issues/new?assignees=&labels=talks-and-workshops%2Cnewsletter&template=give_a_talk.yml&title=%5BTALK%5D+%3Ctitle%3E), which provides links to the resources you will need as well as a checklist for preparing your talk. You will find a range of presentation templates, as well as past presentations, in the Promotion Pack folder on Google Drive. Feel free to download the templates and use the slides as a guide for style and content.
```

Also, do we have a link to the Google Drive folder?

> 

Reviewer: Add here - You can also see all our previous talks on Zenodo by searching for _The Turing Way_. Here is a [link to _The Turing Way_ Community on Zenodo](https://zenodo.org/communities/the-turing-way/?page=1&size=20).

Reviewer: Are you able to add the link to the promotion pack Google folder?

Reviewer 2: An alternative could be to not use Google Drive at all and instead put the promotion pack on zenodo to be downloaded

Reviewer 2: > unlike the pdfs uploaded to zenodo.

I take your point RE this specific case. I want to make a general point about the Zenodo. It is a failure point if people are only uploading PDFs to Zenodo. It can handle multiple file formats and people should be sharing **both** in the Zenodo record (PDF + whatever they created the PDF from). Because then the pptx or whatever would also be clonable and editable.

### Giving a Talk
We ask that you include certain acknowledgments in your talk to make sure contributors are credited. You'll find all the info you need in the presentation templates as well as the GitHub issue checklist.

### After Your Talk

> Keep spreading the word! Please [save your slides as a PDF](https://www.wikihow.com/Convert-Powerpoint-to-PDF) and upload to [Zenodo](https://zenodo.org/), and don't forget to tag "the-turing-way" under Communities.

Reviewer 2: ```suggestion
Keep spreading the word! Please save your slides in their original format and [as a PDF](https://www.wikihow.com/Convert-Powerpoint-to-PDF) and upload to [Zenodo](https://zenodo.org/communities/the-turing-way/), and don't forget to tag "the-turing-way" under Communities.
```

If they upload from this page, the tagging will be done automatically 😉

> Keep spreading the word! Please save your slides in their original format and [as a PDF](https://www.wikihow.com/Convert-Powerpoint-to-PDF) and upload to [Zenodo](https://zenodo.org/communities/the-turing-way/), and don't forget to tag "the-turing-way" under Communities.

Reviewer: ```suggestion
Keep spreading the word! Please archive your slides in their original format and [as a PDF](https://www.wikihow.com/Convert-Powerpoint-to-PDF) and upload to [Zenodo](https://zenodo.org/communities/the-turing-way/), and don't forget to tag "the-turing-way" under Communities.

Please make sure that you credit all of the contributors and presenters of your presentation as authors in the Zenodo upload form. If possible, link to any previous talks where you have used additional slides outside of the provided _The Turing Way_ presentation templates - this can be done in the description part of the upload form.
```

Reviewer 4: ```suggestion
Keep spreading the word! Please archive your slides in two formats: their original (editable) format and as a PDF. If you don't know how to convert PowerPoint slides to PDF, see these [instructions](https://www.wikihow.com/Convert-Powerpoint-to-PDF); following these instruction helps preserve accessibility features for people using screen readers. You can archive your slides on [Zenodo](https://zenodo.org/communities/the-turing-way/), and don't forget to tag "the-turing-way" under Communities.
```

### Preparing Slides
Please create a GitHub issue using the ["Give a talk" issue template](https://github.com/alan-turing-institute/the-turing-way/issues/new?assignees=&labels=talks-and-workshops%2Cnewsletter&template=give_a_talk.yml&title=%5BTALK%5D+%3Ctitle%3E), which provides links to the resources you will need as well as a checklist for preparing your talk. You will find a range of presentation templates, as well as past presentations, in the [Promotion Pack](https://drive.google.com/drive/folders/1mzGmbJkPnP5q1goQesxDc_E5zAPL0eTF?usp=sharing) folder on Google Drive. Feel free to download the templates and use the slides as a guide for style and content. You can also see past talks on _The Turing Way's_ [community page on Zenodo](https://zenodo.org/communities/the-turing-way/?page=1&size=20).

### Giving a Talk
We ask that you include certain acknowledgments in your talk to make sure contributors are credited. You'll find all the info you need in the presentation templates as well as the GitHub issue checklist.

### After Your Talk

> Keep spreading the word! Please archive your slides in two formats: their original (editable) format and as a PDF. If you don't know how to convert PowerPoint slides to PDF, see these [instructions](https://www.wikihow.com/Convert-Powerpoint-to-PDF); following these instruction helps preserve accessibility features for people using screen readers. You can archive your slides on [Zenodo](https://zenodo.org/communities/the-turing-way/), and don't forget to tag "the-turing-way" under Communities.

Reviewer 2: Is there a similar guide for exporting Google Slides to PDF? They are probably the 2 big platforms that we should care about being specific about.

Author: the wikihow article actually covers both, I will update the text to reflect that
